api/services/export-data-2.js

```diff
@@ -163,6 +298,15 @@ class SearchResultReader extends Readable {
     return escapedCsvLine.join(JOIN_COL);
   }
 
+  getDocIds() {
```

Comment:
Did we consider this (the choice of how to resolve IDs we care about) as part of the configuration object above?